EDITING.md

```diff
@@ -0,0 +1,177 @@
+Editing
+=======
+
+Editing specifications is hard!  Here are some partial guidelines and rules-of-thumb.
+
+- [Communicate!](#communicate)
+- [Make sure your change fits](#make-sure-your-change-fits)
+  - ... especially if it's a [semantic change](#semantic-changes)
+- [Use Exploration Reports](#use-exploration-reports)
+- Heed the [stylistic preferences](#stylistic-preferences)
+
+
+Communicate!
+------------
+
+There's lots of ways to communicate with the IPLD team and community.
+
+- IRC/Matrix/Discord: FIXME LINK
+- All our development is in the open on GitHub:
+  - This repo is https://github.com/ipld/specs/ !
+  - Other work exists in other repos the same org: https://github.com/ipld/
+  - GitHub issues and Pull Requests work just fine.
+- We have a (mostly) weekly meeting on Zoom: FIXME LINK
+
+If you'd like to discuss something before proposing a change,
+there's lots of venues that can host a conversation.
+
+Escalating in levels of formality can be a good idea:
+e.g., ask questions on IRC/Matrix/Discord or in an issue first;
+then work on PRs later if an idea seems worth pursuing.
+
+
+Make sure your change fits
+--------------------------
+
+### textual changes
+
+Small textual changes can be made by Pull Request easily.  Go for it!
+
+(Since the amount of effort involved in creating and reviewing this kind of
+change is minimal for both sides, it's easier to just send it than to ask permission.)
+
+### semantic changes
+
+_Editing specifications is **really** hard!_
+
+We welcome everyone's effort in refining the specs to be the best they can be.
+At the same time, be warned that it's a serious investment of energy!
+As shepherds of what we hope is a stable process, we have to be careful and
+deliberate when considering what changes to accept.
+
+A lot of decisions in specifications involve looking out for long-range effects
+and non-obvious implications.  We try to document these considerations,
+but also in many documents try to strike a balance with brevity.
+Hitting a perfect stride with this is arguably objectively not possible;
+it's pretty much guaranteed that in your reading, you'll encounter a document
+that doesn't sufficiently explain all the "why isn't this different" branches.
+
+Specs deserve an extra long think before proposing changes.
+[Chesterton's fence](https://en.wikipedia.org/wiki/Wikipedia:Chesterton%27s_fence)
```

Comment:
I love that you've inserted this reference. Sadly the implications are usually lost on most people who wade straight in on the deep end though.